packages/cursorless-engine/src/runCommand.ts

```diff
@@ -36,7 +44,7 @@ export async function runCommand(
   rangeUpdater: RangeUpdater,
   commandRunnerDecorators: CommandRunnerDecorator[],
   command: Command,
-): Promise<unknown> {
+): Promise<CommandResponse | unknown> {
```

Comment:
Unfortunate that we will have to keep adding to this union as we change our return shape. Would be cool to use type mapping with discriminated union based on input command version number, but prob not worth the trouble. We can revisit if it becomes annoying

Reply:
Agreed